src/state_title_screen.c

```diff
@@ -102,15 +104,15 @@ void state_init_title_screen(void) {
     gGameStatusPtr->creditsViewportMode = -1;
     intro_logos_update_fade();
     titleData = load_asset_by_name("title_data", &titleDataSize);
-    titleDataDst = D_800A0974 = heap_malloc(titleDataSize);
+    titleDataDst = TitleScreen_GfxData = heap_malloc(titleDataSize);
     decode_yay0(titleData, titleDataDst);
     general_heap_free(titleData);
 
-    D_800A0978 = (s32*)(D_800A0974->unk_00 + (s32) D_800A0974);
-    D_800A097C = (s32*)(D_800A0974->unk_04 + (s32) D_800A0974);
-    D_800A0980 = (s32*)(D_800A0974->unk_08 + (s32) D_800A0974);
+    TitleScreen_GfxData_tile = (s32*)(TitleScreen_GfxData->img_tile + (s32) TitleScreen_GfxData);
```

Comment:
tile/tile4b/block will be the `Press Start` text, the copyright notice and the logo.